renderdoc/driver/metal/official/metal-cpp.h

```diff
@@ -5645,29 +5646,41 @@ namespace Private
             "layer");
         _CA_PRIVATE_DEF_SEL(texture,
             "texture");
+        _CA_PRIVATE_DEF_SEL(device,
+            "device");
 
     } // Class
 } // Private
 } // CA
 
 namespace CA
 {
+class MetalLayer : public NS::Referencing<MetalLayer>
```

Comment:
Expand the metal-cpp version of` CA::MetalLayer`: could extend this in the future to replace the ObjC helpers